requirements-dev.txt

```diff
@@ -81,44 +66,25 @@ inflection==0.5.1
     #   pytest-factoryboy
 iniconfig==2.0.0
     # via pytest
-isort==5.13.2
-    # via -r requirements-dev.in
 markupsafe==2.1.5
     # via
     #   -c requirements.txt
     #   werkzeug
-mccabe==0.7.0
-    # via flake8
```

Comment:
Should [mccabe (C90)](https://docs.astral.sh/ruff/rules/#mccabe-c90) be included in linting rules as well?

Reply:
It was agreed that `mccabe` is unnecessary and will be left out.